guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md

```diff
@@ -57,11 +57,11 @@ You must run cron twice: the first time to discover tasks to run and the second
 ## Logging
 System logs provide detailed information for debugging. The following attributes apply to Magento cron logs:
 
--   All exceptions from cron jobs are logged in `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
+-   All exceptions from cron jobs are logged by `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
 
 -   Failed jobs with `ERROR` and `MISSED` statuses are logged to the `<your Magento install dir>/var/log` directory.
 
--   Jobs with an `ERROR` status are always logged as `CRITICAL` in the `<your Magento install dir>/var/log/exception.log` directory.
+-   Jobs with an `ERROR` status are always logged as `CRITICAL` in `<your Magento install dir>/var/log/exception.log`.
 
 -   Jobs with a `MISSED` status are logged as `INFO` in the `<your Magento install dir>/var/log` directory (developer mode only).
```

Comment:
Thanks for the contribution @keithbentrup. Please apply your changes to lines 62 and 66 to maintain consistency and I'll merge this PR.

Reply:
@jeff-matthews I don't understand your request. I'm pointing out in my request that a file is being called a directory. I don't know which file `MISSED` are being logged in, but I assume at least the directory is correct.

Reply:
What I mean is, if your change drops "directory" from one bullet item, you should also drop "directory" from the preceding and following bullet items to maintain the same sentence structure and promote consistency across all three bullets. 

For example:

>-   Failed jobs with `ERROR` and `MISSED` statuses are logged to **the** `<your Magento install dir>/var/log` **directory**.
>-   Jobs with an `ERROR` status are always logged as `CRITICAL` in **the** `<your Magento install dir>/var/log/exception.log` **directory**.
>-   Jobs with a `MISSED` status are logged as `INFO` in **the** `<your Magento install dir>/var/log` **directory** (developer mode only).

Change to:

>-   Failed jobs with `ERROR` and `MISSED` statuses are logged **to** `<your Magento install dir>/var/log`.
>-   Jobs with an `ERROR` status are always logged as `CRITICAL` **in** `<your Magento install dir>/var/log/exception.log`.
>-   Jobs with a `MISSED` status are logged as `INFO` **in** `<your Magento install dir>/var/log` (developer mode only).

Reply:
@jeff-matthews Please reread the original, my change, and follow-up comment, and consider what I've written. 

I'm merely updating the text to reflect what is currently there. The original text is calling a file -> a directory. That's clearly wrong. I don't know how else to explain it.

If I had more information about which specific file MISSED statuses are logged to, I would include it.

Reply:
@keithbentrup: My suggestions supplement your improvements. It'd be nice if you would go ahead and make those changes so we can make all improvements in a single PR instead of me merging this one and opening a new one to make the supplemental changes.